apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -72,7 +75,10 @@ public abstract partial class BigtableClient
             var channelOptions = new[]
             {
                 new ChannelOption(ChannelOptions.MaxSendMessageLength, -1),
-                new ChannelOption(ChannelOptions.MaxReceiveMessageLength, -1)
+                new ChannelOption(ChannelOptions.MaxReceiveMessageLength, -1),
+
+                // TODO: Figure out if there's a good way to test this
+                new ChannelOption(ChannelOptions.PrimaryUserAgentString, $"cbt-csharp/v{versionAttribute.InformationalVersion}")
```

Comment:
Cache this string in a single place, perhaps?

Reply:
Done

Reply:
Oops, forgot to actually run it.

Reply:
(I forgot to actually get the version string from the attribute in the cached string, so it used the type name)